pkg/option/config.go

```diff
@@ -1039,6 +1039,16 @@ func getIPv4Enabled() bool {
 	return viper.GetBool(EnableIPv4Name)
 }
 
+func getIPv6Enabled() bool {
+	if viper.GetBool(EnableIPv6Name) {
```

Comment:
To avoid calling `viper.GetBool` twice, I would change the code to:

```
enableIPv6 := viper.GetBool(EnableIPv6Name)
if enableIPv6 {
	if _, err := os.Stat("/proc/net/if_inet6"); os.IsNotExist(err) {
		return false
	}
}
return enableIPv6
```